.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: ['2.4', '2.5', '2.6', '2.7']
```

Comment:
How do you feel about running this against 2.7, 3.0, 3.1, and 3.2 instead. I think everything pre-2.7 is EOL now

Reply:
That's a good call out. I went initially just for parity with the Travis CI setup, but perhaps let's see if they pass in supported versions.  Will check this out.

Reply:
Updated to test on Ruby 2.7, 3.0, 3.1 and 3.2.  Also updated to test Rails 6.1 and Rails 7.0. Removed testing in Rails 5.x the the exception of Rails 5.2 (under Ruby 2.7).

Considered updating the gemspec as well to bump from Rails 5.0 to Rails 5.2 as the minimum supported Rails version, but didn't know if that should be a separate PR @brimatteng?

Reply:
No strong feelings about it. It seems like it makes sense here